src/frontend/test_runner/tests/testdata/my_test.yaml

```diff
@@ -0,0 +1,8 @@
+- sql: |
```

Comment:
This file will be removed, meant for sanity check. The test case is already part of the test suite.